src/Stratis.Bitcoin.Features.PoA/PoAFeature.cs

```diff
@@ -94,11 +94,7 @@ public override Task InitializeAsync()
                 this.votingManager.Initialize();
             }
 
-            if (this.federationManager.IsFederationMember)
-            {
-                // Enable mining because we are a federation member.
-                this.miner.InitializeMining();
-            }
+            this.miner.InitializeMining();
```

Comment:
I just realized the feature will also register the miner, do we not want to separate the miner to its own feature?

Then nodes that are not fed members can run a daemon that is not a miner (a suggestion to consider for another PR)

Reply:
After introduction of the voting system we don't know if we are a miner or not before we sync. So having miner in the same feature makes sense.

Reply:
In that case users that are miners will know  in advance and run a deamon with the miner

Reply:
I see no need for such complication. 

PoA miner is a small class and is doing nothing in case node is not a fed member. 
So I see no benefit in moving it in another feature.